README.md

```diff
@@ -23,13 +23,12 @@
 ### Dependencies
 ```
 git clone https://github.com/iojs/website.git
-npm install -g gulp
 npm install
 ```
 
 ### Local Development
 ```
-gulp
+npm run build
```

Comment:
@Fishrock123  @KenanY 
Thanks.
Just using `./node_modules/.bin/gulp` on `"build": "gulp"` in npm-scripts.
This is the way not to install global gulp :smiley: